### PR TITLE
Resolves #324. Add characters classmethod to lorem provider

### LINKED ISSUE #324: Add ability to generate lorem characters without punctuation
Sometimes I want to generate a string of characters of a specific length without any punctuation or capitalization but the lorem provider currently does not allow for this.

### PR DESCRIPTION
**Pull request by gwongz**
Sometimes I want to generate a string of characters of a specific length
without any punctuation or capitalization. Adding a characters() method
allows for this. It defaults to a 255-character, mixed case string but
the method can also be called with params that specify length and
whether to exclude uppercase characters.

### CODE REVIEW

**Comment by fcurella** [comment]
I don't think this should go into the `lorem` provider, as it's not providing filler text. I'd rather have it as a replacement for `pystr` inside the `Python` provider.

File: faker/providers/lorem/__init__.py [hunk 1]
```diff
@@ -6,9 +6,24 @@
 
 class Provider(BaseProvider):
     @classmethod
+    def characters(cls, max_nb=255, variable_case=True):
```
Comment (fcurella) [inline faker/providers/lorem/__init__.py]: I don't think `variable_case` is necessary`. You could achieve the same by simply calling `Provider.characters().lower()` (or `.upper()`, or `.title()`)

**Comment by gwongz** [comment]
Thanks for the feedback @fcurella. I moved this from `lorem` to the `Python` provider.

**Comment by fcurella** [comment]
Thank you @gwongz , Nice work! :sparkles:

**Pull request merged**